cleverpush/src/main/java/com/cleverpush/database/TableBannerTrackEvent.java

```diff
@@ -0,0 +1,120 @@
+package com.cleverpush.database;
+
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+@Entity(tableName = "TableBannerTrackEvent")
+public class TableBannerTrackEvent {
+
+  @PrimaryKey(autoGenerate = true)
+  private int id;
+  @ColumnInfo(name = "banner_id")
+  private String banner_id;
+  @ColumnInfo(name = "track_event_id")
+  private String track_event_id;
+  @ColumnInfo(name = "relation")
+  private String relation;
+  @ColumnInfo(name = "property")
+  private String property;
+  @ColumnInfo(name = "value")
+  private String value;
+  @ColumnInfo(name = "from_value")
+  private String from_value;
+  @ColumnInfo(name = "to_value")
+  private String to_value;
+  @ColumnInfo(name = "count")
+  private int count;
+  @ColumnInfo(name = "created_date_time")
+  private String created_date_time;
+  @ColumnInfo(name = "updated_date_time")
+  private String updated_date_time;
+
+  public int getId() {
+    return id;
+  }
+
+  public void setId(int id) {
+    this.id = id;
+  }
+
+  public String getBanner_id() {
+    return banner_id;
+  }
+
+  public void setBanner_id(String banner_id) {
```

Comment:
banner_id can not be camelCase ? Same for below methods.

Reply:
@gbhrdt Only need to change in getter-setter methods or also in the column name?

Reply:
only getter-setter